lib/xlsx/xform/style/styles-xform.js

```diff
@@ -82,6 +82,8 @@ class StylesXform extends BaseXform {
     // add default fills
     this._addFill({type: 'pattern', pattern: 'none'});
     this._addFill({type: 'pattern', pattern: 'gray125'});
+
+    this.weakMap = new Map()
```

Comment:
Nice catch! Quick question - any good reason not to use a WeakMap here?

Reply:
No reason. i'll change it

Reply:
Changed